docs/funding/google-season-of-docs-2023.md

```diff
@@ -0,0 +1,134 @@
+
+# About your project
+
+## Your project’s problem
+
+<!---
+Tell us about the problem your project will help solve. Why is it important to your organization or project to solve this problem?
+-->
+
+The JupyterLab accessibility team has been concentrating its efforts on accessibility remediation through work on interface audits, testing events, improvements to UI/UX and automated testing. Due to the limited number of resources and the prioritization of such remediation work, documenting accessibility features has been a secondary priority. As more accessibility-focused improvements are added, it is becoming increasingly necessary that we provide a dedicated focus on adding user accessibility documentation for JupyterLab.
+
+Through this project, we aim to create documentation primarily targeted at anyone with a temporary, situational, or permanent disability that wants or needs to use JupyterLab. We want to provide the JupyterLab community with a comprehensive guide about the accessibility features present in the application. For example, developing documentation that answers the questions “What can and can’t I do in JupyterLab with accessibility accommodations?” and “How can I adjust JupyterLab to work better for me with assistive technology?”
+
+## Your project’s scope
+
+<!---
+Tell us about what documentation your organization will create, update, or improve. If some work is deliberately not being done, include that information as well. Include a time estimate, and whether you have already identified organization volunteers and a technical writer to work with your project.
+-->
+
+This project includes:
+
+- Creation of an accessibility section in JupyterLab documentation containing default accessibility settings and customizations inspired by https://code.visualstudio.com/docs/editor/accessibility
+- Create user-focused documentation that summarizes the state of JupyterLab areas that have been audited for accessibility.  This includes an approachable report on keyboard navigation and 400% zoom audits results.
+
+As a stretch goal, we would like to include:
+
+- An authoring guide focused on best practices to generate accessible notebooks.
+
+Work that is out of the scope of this project includes:
+
+- Creation of developer-focused documentation
+- Tutorials and workshops to present in conferences
+- Live streams and social media engagement
+
+The developer that will be mentoring this project has previously mentored successful GSoD in another project last year and has been involved in the accessibility remediation work for JupyterLab. We don’t have a technical writer for this project, but we intend to start the  application and selection process once we receive the acceptance by Google Season of Docs.
```

Comment:
Do we need to name the mentor?

Reply:
Last year we didn't named anyone in the proposal.